src/graphql/admin/schema.graphql

```diff
@@ -1,29 +1,72 @@
 type Query {
   allLevels: [AccountLevel!]!
-  userDetailsByPhone(phone: Phone!): UserDetails!
-  userDetailsByUsername(username: WalletName!): UserDetails!
+  userDetailsByPhone(phone: Phone!): MerchantUser!
+  userDetailsByUsername(username: WalletName!): MerchantUser!
 }
 
 enum AccountLevel {
   ONE
   TWO
 }
 
-type UserDetails {
+type MerchantUser {
```

Comment:
what is a merchant user? is there any difference with normal users?

Reply:
Yes. This is used in the admin schema only. It exposes the leve/status/map-info.

`MerchantUser` is a bad name though. I renamed it to `UserForAdmin` (locally)